### ISSUE TITLE
How to "edit in place"?

### ISSUE THREAD

**Issue by caedmon-kitty** [sedwards2009/extraterm#437]
extraterm 0.75
The guide states "Or you can edit the command output directly in place ..." but give no information on how to do so.

How do I go to a previous frame and edit the output in place?

**Comment by sedwards2009** [comment]
Sorry, the guide is out of date. I've updated parts of it recently but not that. The editing in place feature isn't in the current Qt based versions.

The last version to have that feature is https://github.com/sedwards2009/extraterm/releases/tag/v0.59.4 . based on Electron.

**Comment by cmprmsd** [comment]
Will this by any chance come to the Qt version, too? 👀 ☺️

**Comment by pcause** [comment]
bump.  seems like a cool but also needed feature to make frames fully useful

**Comment by cmprmsd** [comment]
@sedwards2009 
Would you like to have assistance on this issue? Maybe you just decided you don't use it.
Curently only Warp Terminal has this feature afaik.

**Comment by sedwards2009** [comment]
@cmprmsd You are right. It is not a high priority for me at the moment. You are welcome to try yourself though.
